infra/nnrt-micro/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(nnrt-micro)
```

Comment:
@lemmaa , do you have any prefered name?
existing ones are `nncc`, `nnfw` from historical reasons.
I thought of `onert-micro` as our runtime name is `OneRT`

Reply:
>to follow existing `nnfw` and name it as `nnfw-lite` seems OK.
>root name uses `runtime-lite` so using `lite` instead of `micro` would be better

As @seanshpark commented, `nnfw` comes from historical reason. And there is no policy for naming in `/infra/(project)`. So IMO, it's no problem to use anything `nnrt-micro`, `nnfw-lite`, or `nnrt-lite`. 

>we can use `onert-lite` as our runtime has `onert` for its name

It looks runtime under `/runtime-lite/` has no relationship with current runtime, so it's better to use independent name with `onert`. 

BTW, why this PR's title and title's label uses naming `runtime-micro`, not `runtime-lite`?

Reply:
Through the naming history of `nnfw`, `NN Runtime`, etc., we are using `onert` officially. If you discard the name `luci-*`, how about using `onert-*` for project naming consistency?

The beginning of `luci-micro` was the response of `tensorflow micro`. The correspondence between `onert` and `tensorflow lite(=tflite)` may be a little awkward, but the correspondence between `onert-lite` and `tensorflow micro` is more confusion for me. Wouldn't it be better to use `onert-micro`, personally?

Reply:
@seanshpark, @lemmaa , @hseok-oh,

Choose `onert-micro`, what do you think about it?

Reply:
> BTW, why this PR's title and title's label uses naming `runtime-micro`, not `runtime-lite`?

typo, fixed it, thanks :)

Reply:
Good to me :)